becca-new-app/src/App.js

```diff
@@ -1,18 +1,20 @@
-import React from 'react';
-
+import * as React from 'react';
+todoList[
```

Comment:
something is missing here. have you tried running your app locally with `npm start`? seems like it's failing due to a couple of things.